packages/core/src/components/Dropdown/Dropdown.jsx

```diff
@@ -96,6 +97,7 @@ export class Dropdown extends React.PureComponent {
           {...selectProps}
         >
           {optionElements}
+          {children}
```

Comment:
This is one of those situations where we want either `options` or `children`, but theres no good way to have that be enforced with Proptypes. Because we almost always want `options` to be defined, I left `options` as a required prop, while `children` is optional.